dist/datamaps.all.js

```diff
@@ -621,7 +621,7 @@
           // Deep copy if property not set
           if (obj[prop] == null) {
             if (typeof source[prop] == 'function') {
-              obj[prop] = source[prop].bind({});
+              obj[prop] = source[prop];
```

Comment:
@magic890 calling `options.setProjection.apply(this, [options.element, options] );` on line 692 was getting ignored, `setProjection` was getting called but `this` was just `{}`.